### PR TITLE
adding task to tag bugzillas

### PR DESCRIPTION
**Pull request by adhil0**
- Celery task to add "telco" and/or "telco:case" to a relevant bugzilla's internal whiteboard

### CODE REVIEW

File: dashboard/src/t5gweb/taskmgr.py [hunk 1]
```diff
@@ -129,4 +137,23 @@ def cache_data(data_type):
...
+    logging.warning("getting bugzillas")
+    bz_url = "bugzilla.redhat.com"
+    cfg = t5gweb.set_cfg()
+    bz_api = bugzilla.Bugzilla(bz_url, api_key=cfg['bz_key'])
+    bugs = libtelco5g.redis_get('bugs')
+    logging.warning("tagging bugzillas")
+    for case in bugs:
+        for bug in bugs[case]:
+            bz = bz_api.getbug(bug['bugzillaNumber'])
+            if "telco" not in bz.internal_whiteboard.lower():
```
Comment (dcritch) [inline dashboard/src/t5gweb/taskmgr.py:154]: I'm wondering... can we get the internal whiteboard stuff when we cache the BZs? that way we don't need to call the bz API on every bug.

Comment (adhil0) [reply dashboard/src/t5gweb/taskmgr.py]: Yeah, I think that would better! I just added the internal_whiteboard to the cache

Comment (dcritch) [reply dashboard/src/t5gweb/taskmgr.py]: Awesome, thanks!

**Comment by dcritch** [comment]
One other nit: we have a bunch of things scheduled that happen on the hour and it is starting to build up (maybe we can add some randomness to this on scheduler startup later on) but for now, can you change the interval to run at a time that is not the top of an hour? 37, 23, 48, whatever. something 'random' for now

**Comment by adhil0** [comment]
Fixed!

**Comment by dcritch** [comment]
Perfect. thanks!

**Pull request merged**